### DISCUSSION TITLE
Abuse and Postmaster routes for return path domain

### DISCUSSION THREAD

**Discussion by pcarr-m3** [postalserver/postal#2715] [Help with using Postal]
Postal sends email from the return path domain psrp.<domain>, however for feedback loop reporting the routes abuse@psrp.<domain> and postmaster@psrp.<domain> need to exist.  There doesn't seem to be a way to create these routes though.

We are trying to get our Postal server certified with Validity and this issue is currently holding up the certification.  Has anyone been able to find a workaround to this?  Thanks

**Comment by willpower232** [comment]
It sounds like you might need to use the sender header to include a different domain name in the email for one you can create abuse and postmaster addresses for? I don't think you'll be able to make Postal handle those two.
